docs/post/memcards.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Memory Cards"
+date: 2024-03-20
+summary: "This guides shows you how to import save data to your virtual memory card"
+draft: false
+toc: true
+sidebar_position: 4
+---
+
+This section shows you how to import save data to your virtual memory card
+
+## General
+
+Most save files for PlayStation 2 games during the console's lifecycle were extracted from consoles using the various cheat devices which were sold at the time. Some examples are Action Replay Max, Code Breaker, and GameShark. However, because these companies were competing and did not want any other companies to be able to use saves extracted with their devices, the saves were repackaged into proprietary formats and encrypted. To use these save files on PCSX2, they must be decrypted and unpackaged by specialized software.
+
+## What is MyMC?
+
+MyMC is software designed to interact with PlayStation 2 memory card images (.ps2) as was used by PCSX2. It allows you to add, remove, as well as export save data.
```

Comment:
"as was used by" should be present tense plural, "as are used by".

"add, remove, as well as export" should be either
1) "add and remove, as well as export"
2) "add, remove, and export"